fiasco/ion.py

```diff
@@ -92,6 +92,12 @@ def __radd__(self, value):
     def transitions(self):
         return Transitions(self._elvlc, self._wgfa)
 
+    @property
+    @needs_dataset('elvlc', 'wgfa')
+    def non_twophoton_transitions(self):
+        all_transitions = self.transitions
+        return all_transitions[~all_transitions.is_twophoton]
```

Comment:
Somehow I just now noticed this but how exactly does this work? There is no __getitem__ implemented on Transitions so what is it slicing here exactly?

Reply:
Haha oh dear I'm not sure. Let me give this another go (and add tests!!!)

Reply:
Not opposed to having getitem on Transitions btw. That could be interesting.